### PR TITLE
verify some code

### PR DESCRIPTION
**Pull request by minhhieu1612**

### CODE REVIEW

**Comment by uiterzeronum** [comment]
please test your feature once again on your local environment

File: shopping/src/database/connection.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = async() => {
         await mongoose.connect(DB_URL, {
             autoIndex: true
         });
-        console.log('Db Connected');
+        console.log('Db Connected!!');
```
Comment (uiterzeronum) [inline shopping/src/database/connection.js:11]: please explain this change

Comment (minhhieu1612) [reply shopping/src/database/connection.js:11]: this code is to test the merge & review process

Comment (uiterzeronum) [reply shopping/src/database/connection.js:11]: thanks, I got your idea

**Review by uiterzeronum — APPROVED** [review]
Everything look okay right now, so I approved this PR

**Pull request merged**